services/gacha_log_service.py

```
@@ -0,0 +1,98 @@
+            end_ids[gacha_type] = max(end_ids[gacha_type], item_id)
+    
+    return end_ids
+
+
+def upload_gacha_log(user_id, uid, items):
+    """上传祈愿记录"""
+    # 查找是否已有该用户和UID的祈愿记录
+    existing_log = client.ht_server.GachaLog.find_one({"user_id": user_id, "Uid": uid})
+    
+    if existing_log:
+        # 已有数据，合并新旧数据（按Id去重）
```

Ruff (0.14.10)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)